packages/wrangler/src/dev/local.tsx

```diff
@@ -83,9 +83,29 @@ function useLocalWorker({
 
       const scriptPath = realpathSync(bundle.path);
 
+      // the wasm_modules/text_blobs/data_blobs bindings are
+      // relative to process.cwd(), but the actual worker bundle
+      // is in the temp output directory; so we rewrite the paths to be absolute,
+      // letting miniflare resolve them correctly
+
+      // wasm
       const wasmBindings = { ...bindings.wasm_modules };
+      for (const [name, filePath] of Object.entries(wasmBindings)) {
+        wasmBindings[name] = path.join(process.cwd(), filePath);
+      }
```

Comment:
I think this results in copying over the properties twice. Once for the `...bindings.wasm_modules` spread operation, and then again for the path mapping.
Could we just do it in one go?

```suggestion
      const wasmBindings = { };
      for (const [name, filePath] of Object.entries(bindings.wasm_modules)) {
        wasmBindings[name] = path.join(process.cwd(), filePath);
      }
```

Reply:
Or am I missing something here?

If you are happy with this approach, then the same is true of the ones below.

Reply:
yeah this is fine.